packages/admin-ui/src/checkbox/checkbox-input.tsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useRef } from 'react'
+import type { CheckboxOptions as AriakitCheckboxOptions } from 'ariakit/checkbox'
+import { Checkbox as AriakitCheckbox } from 'ariakit/checkbox'
+import { createComponent, useElement } from '@vtex/admin-ui-react'
+import { useForkRef } from '@vtex/admin-ui-hooks'
+
+import * as style from './checkbox.style'
+
+export const CheckboxInput = createComponent<
+  typeof AriakitCheckbox,
+  CheckboxInputOptions
+>((props) => {
+  const { error = false, ref: htmlRef, state, ...htmlProps } = props
+
+  const ref = useRef<HTMLInputElement & any>(null)
```

Comment:
I had to add `any` to be able to use `ariaChecked`, because the build complains about `ariaChecked` is not in the HTMLInputElement interface